build.gradle

```diff
@@ -65,6 +72,24 @@ test {
 	}
 }
 
+task cleanTestData(type: Delete) {
```

Comment:
Maybe add this as a dependency of the `clean` task?

Reply:
Since `clean` just deletes the entire `build` directory (where I store the generated resources as well), that's not needed.